examples/ASDKgram/Podfile

```diff
@@ -1,6 +1,8 @@
 source 'https://github.com/CocoaPods/Specs.git'
 platform :ios, '8.0'
 target 'Sample' do
-	pod 'Texture/IGListKit', :path => '../..'
-	pod 'Texture/PINRemoteImage', :path => '../..'
+  pod 'PonyDebugger', :git => 'https://github.com/nguyenhuy/PonyDebugger.git', :branch => 'master'
```

Comment:
Once we have completely forked PonyDebugger and publish a TextureDebugger pod, this will be unnecessary.